frontend/src/components/AddMember/index.js

```diff
@@ -3,101 +3,181 @@ import {
   Hero,
   Button,
   Input,
-  Label,
   Select,
   RadioList
 } from "@code4ro/taskforce-fe-components";
 import SidebarLayout from "../SidebarLayout";
 import "./AddMember.scss";
-const AddMember = () => {
-  //TODO: change with api endpoins responses when they are available
-  const relationshipOptions = [
-    { text: "", value: "" },
-    { text: "Bunică", value: "1" }
-  ];
+class AddMember extends React.Component {
```

Comment:
All our components should be function components.

Reply:
Thanks for the hint, @surdu! Do you have a place to start in understanding how to apply state so that we can take advantage of the auto re-rendering from React when the state is changed? Or should we rely on plain ol' JS show/hide for that and ditch the state?

Reply:
Besides all the components in our app you might also look for examples in [our shared components library](https://github.com/code4romania/taskforce-fe-components/tree/master/src/components)

Reply:
Updated.